master/buildbot/reporters/github.py

```diff
@@ -94,10 +95,16 @@ def createStatus(self,
         if context is not None:
             payload['context'] = context
 
-        return self._http.post(
+        if self.verbose:
```

Comment:
I guess this is a separate feature which belongs to a separate PR. I think this would be better to put it in HttpBase reporter